ffi/examples/read-table/CMakeLists.txt

```diff
@@ -18,11 +18,11 @@ add_test(NAME read_and_print_basic_partitioned COMMAND ${TestRunner} ${DatPath}/
 
 if(WIN32)
   set(CMAKE_C_FLAGS_DEBUG "/MT")
-  target_link_libraries(read_table PUBLIC ws2_32 userenv bcrypt ntdll)
+  target_link_libraries(read_table PUBLIC ws2_32 userenv bcrypt ncrypt crypt32 secur32 ntdll RuntimeObject)
```

Comment:
I'd like to also point out a lot of these are because the underlying rust libs aren't linking these statically when they build. I'm not entirely sure how to investigate or remediate this, but I am just mentioning I noticed this when figure out what to link.